aws/onboarding/byok8s/manual-steps/README.md

```diff
@@ -37,33 +37,33 @@ This documentation outlines the steps to manually create an EKS cluster and asso
         - **Type**: HTTPS
         - **Protocol**: TCP (Prefilled)
         - **Port range**: 443 (Prefilled)
-        - **Source**: Custom, `sg-rubrik-node-security-group` (Replace with actual Node Security Group ID created above from dropdown)
+        - **Source**: Custom, `rubrik-node-security-group` (Replace with actual Node Security Group ID created above from dropdown)
         - **Description**: Inbound traffic from worker nodes
 2. **Edit outbound rules for** `rubrik-cluster-security-group`.
     - **Add the following rule** and save:
         - **Type**: Custom TCP
         - **Protocol**: TCP (Prefilled)
         - **Port range**: `1025 - 65535`
-        - **Source**: Custom, `sg-rubrik-node-security-group` (Replace with actual Node Security Group ID created above from dropdown)
+        - **Source**: Custom, `rubrik-node-security-group` (Replace with actual Node Security Group ID created above from dropdown)
         - **Description**: Outbound traffic to worker nodes
-3. **Edit inbound rules for** `sg-rubrik-node-security-group`.
-    - **Add the following rule**:
+3. **Edit inbound rules for** `rubrik-node-security-group`.
+    - **Add the following rule**and save:
```

Comment:
A customer can also add all the rules in one go and then save. Should we drop the `and save` part from everywhere?

Reply:
Updated